schema/360-giving-schema.json

```diff
@@ -726,7 +726,7 @@
             "ZM",
             "ZW"
           ],
-          "description": "The ISO Country Code of the location of this activity.",
+          "description": "The 2-character ISO Country Code of the location of this activity. Taken from [ISO-3166-1-alpha-2](https://www.iso.org/iso-3166-country-codes.html)",
```

Comment:
It was agreed to use link https://en.wikipedia.org/wiki/ISO_3166-1_alpha-2 rather than ISO link because ISO is not open.